integrations/zendesk/src/client.ts

```diff
@@ -141,6 +141,30 @@ class ZendeskApi {
     const { data } = await this.client.get<{ user: ZendeskUser }>(`/api/v2/users/${userId}.json`)
     return data.user
   }
+
+  public async createArticleWebhook(webhookUrl: string, webhookId: string): Promise<void> {
+    await this.client.post('/api/v2/webhooks', {
+      webhook: {
+        endpoint: `${webhookUrl}/article-event`,
+        http_method: 'POST',
+        name: `bpc_article_event_${webhookId}`,
```

Comment:
note: webhookId added to account for case when several bots connected to the same zendesk account